serverless/pages/what-is-elasticsearch-serverless.asciidoc

```diff
@@ -1,32 +1,61 @@
-////
-To be rewritten/refined
-////
+// ℹ️ THIS CONTENT IS RENDERERED ON THE index-serverless-elasticsearch.asciidoc PAGE
+// Use the id <<what-is-elasticsearch-serverless>> to link to this page
 
 // :description: Build search solutions and applications with {es-serverless}.
 // :keywords: serverless, elasticsearch, overview
 
 preview:[]
 
-{es} allows you to build custom applications. Whether you have structured or unstructured text, numerical data, or geospatial data, {es} can efficiently store and index it in a way that supports fast searches.
-
-.Understanding {es-serverless}
-[IMPORTANT]
+[TIP]
 ====
-Refer to <<elasticsearch-differences,Serverless differences>> and <<elasticsearch-technical-preview-limitations,Technical preview limitations>> for important details, including features and limitations specific to {es-serverless}.
+If you haven't used {es} before, you might want to learn the basics in the https://www.elastic.co/guide/en/elasticsearch/reference/current/elasticsearch-intro.html[main {es} documentation].
 ====
 
-[discrete]
-== Get started
+{es} is an open source distributed search and analytics engine, scalable data store, and vector database. It’s optimized for speed and relevance on production-scale workloads. Use {es} to search, index, store, and analyze data of all shapes and sizes.
```

Comment:
this feels kinda weird because ES is also the foundation of obs and sec projects. could we reframe the ES project type as the "vanilla/search" option?

something like: 

```
{es} is one of the three available project types on <<intro,{serverless-full}>>.

This project type lets you use the core functionality of {es}: Searching, indexing, storing, and analyzing data of all shapes and sizes.
```

and then let the es basics page take care of the "what is ES" question? 

"lets you use" might feel restrictive - something like "is built for" "is optimized for"?

Reply:
nods in agreement, but on the "foundation of obs and sec projects" point I couldn't find a clean way to incorporate that info here but I'm also not too worried about that really

Reply:
cleared up intro in https://github.com/elastic/docs-content/pull/193/commits/040ebc51d3df2361e6788cd8c6f07f9a32a8c8b8

Reply:
re: the foundational aspects we can revisit if it seems like people are creating additional projects in error